kalite/distributed/static/js/distributed/exercises/perseus-helpers.js

```diff
@@ -1,10 +1,23 @@
+var _ = require("underscore");
+var Backbone = require("base/backbone");
+var $ = require("base/jQuery");
+
+require("../perseus/lib/es5-shim.js");
+global.React = require("react/addons");
```

Comment:
global is equivalent to the 'window' object - but browserify uses the 'global' object internally in order to maintain full compatibility with node.js (which uses global on the server side).